win32ss/user/ntuser/simplecall.c

```diff
@@ -798,9 +798,9 @@ NtUserCallHwndParam(
             UserRefObjectCo(pWnd, &Ref);
 
             if (pWnd->head.pti->ppi == PsGetCurrentProcessWin32Process() &&
-                pWnd->cbwndExtra == DLGWINDOWEXTRA &&
```

Comment:
Please keep this check (could be "improved" to `pWnd->cbwndExtra >= DLGWINDOWEXTRA` because it is possible to ask for more than this size) for compatibility with Windows, that is such that when the size is larger than `DLGWINDOWEXTRA` it indicates it's a dialog (even if the Win32 dialog data structure is not stored in there).

Check https://devblogs.microsoft.com/oldnewthing/20031113-00/?p=41843 and
https://stackoverflow.com/questions/866692/if-you-add-extra-data-space-to-a-dialogbox-class-be-accessed-by-getwindowlongptr

Reply:
"That's not what Windows does!"™

Reply:
I'm *not* saying that they store the extra dialog data in that space (they don't); I'm just saying there's a check of the cbwndExtra size just to *validate* it's a dialog.

Reply:
Changed accordingly.